src/test/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedCronTabTest.java

```diff
@@ -41,4 +42,30 @@ public void ctor_happyPath() throws Exception {
 
 	}
 
+	@Test
+	public void param_value_with_percent_sign() throws Exception {
+		String cron = "* * * * *";
+		String params = "one=onevalue;two=10%";
+		String line = cron +" %" + params;
+		Map<String, String> parameters = Maps.newHashMap();
+		parameters.put("one", "onevalue");
+		parameters.put("two", "10%");
+		CronTab testCronTab = new CronTab("* * * * *");
+		ParameterizedCronTab testObject = new ParameterizedCronTab(testCronTab, parameters);
+
+		ParameterizedCronTab parameterizedCronTab = ParameterizedCronTab.create(line, 1, Hash.from(line), null);
+		assertEquals(parameters, parameterizedCronTab.getParameterValues());
+
+	}
+
+	@Test
+	public void with_no_params_seperator() throws Exception {
```

Comment:
spelling should be **separator** not seperator

great PR otherwise - I came here looking for a way to use timezone :)

Reply:
Hi @shbhardwaj 
Did you have chance to look at above comment?
Thanks!

Reply:
spelling fixed